onboarding/src/Components/QA/QAPage.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Navbar from "../Navbar";
+import LeftMenu from "../LeftMenu";
+import PageAddressBar from "../PageAddressBar";
+import QAList from "./QAList";
+
+const QAPage = () => {
+  return (
+    <div className="app">
+      <header className="app-header app-header-dark">
+        <Navbar />
```

Comment:
I know ... ;-)
`<Navbar loggedUser={ loggedUser } />`
Same for lines
13 and 18.